.codecov.yml

```diff
@@ -0,0 +1,37 @@
+# To validate:
+#   cat codecov.yml | curl --data-binary @- https://codecov.io/validate
+
+codecov:
+  notify:
+    require_ci_to_pass: no
```

Comment:
Have we tried this change with ci_to_pass yes ?

Reply:
Yes, but then we won't get the coverage update until after the integration tests have been completed. Since they are gated, and then also take around 3.5h, it will just delay the report a lot.

Reply:
Got it.. this is an useful change, approving it.